### PR TITLE
Validation: Removed Absolute Links

### PR DESCRIPTION
**Pull request by dereklh7**
## Category

- [x] Content fix
- [ ] New article

## Related issues

This PR is part of the Validation cleanup efforts.

Removed absolute links.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[4892e6a](https://github.com/dereklh7/sp-dev-docs/commits/4892e6ae38d709790e99dda50d5929b661e7eae9)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/spfx/publish-to-marketplace-after-publishing.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/publish-to-marketplace-after-publishing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/publish-to-marketplace-after-publishing?branch=pr-en-us-8172) | [Details](#user-content-3595a6840e3d6c4f051122128d68d2b1-047a262e)

<a id="3595a6840e3d6c4f051122128d68d2b1-047a262e"></a>
### [docs/spfx/publish-to-marketplace-after-publishing.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/publish-to-marketplace-after-publishing.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206132313276915-8172/BuildReport?accessString=f4936631c022263a85f2871d68ad41e09b616e774e3e8904384fc4e6390b3c41).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[69ef6dc](https://github.com/dereklh7/sp-dev-docs/commits/69ef6dc7e7fb0cd88f0fee67d5154c70eaf28823)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/spfx/publish-to-marketplace-after-publishing.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/publish-to-marketplace-after-publishing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/publish-to-marketplace-after-publishing?branch=pr-en-us-8172) | [Details](#user-content-3595a6840e3d6c4f051122128d68d2b1-0c0e3291)
[docs/spfx/use-aadhttpclient.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-aadhttpclient.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/use-aadhttpclient?branch=pr-en-us-8172) | [Details](#user-content-38dcf3b52fdcdc8ec41d348a773a9911-0c0e3291)
[docs/spfx/use-theme-colors-in-your-customizations.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-theme-colors-in-your-customizations.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/use-theme-colors-in-your-customizations?branch=pr-en-us-8172) | [Details](#user-content-d673714d2bccd161432f37861698a99c-0c0e3291)

<a id="3595a6840e3d6c4f051122128d68d2b1-0c0e3291"></a>
### [docs/spfx/publish-to-marketplace-after-publishing.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/publish-to-marketplace-after-publishing.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="38dcf3b52fdcdc8ec41d348a773a9911-0c0e3291"></a>
### [docs/spfx/use-aadhttpclient.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-aadhttpclient.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="d673714d2bccd161432f37861698a99c-0c0e3291"></a>
### [docs/spfx/use-theme-colors-in-your-customizations.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-theme-colors-in-your-customizations.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206132318059291-8172/BuildReport?accessString=99eba683726f3e5218e231b5015eb4a0b64cb205c25db65bbaa0ba52ece9e550).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/spfx/publish-to-marketplace-after-publishing.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ SharePoint Online allows you to create a tenant App Catalog where you centrally
 
 Applications that you acquire from the Marketplace, you can install only in tenant App Catalogs. After installing the application, you can decide whether you want to deploy it globally and make it available to all sites in your tenant or if you want to deploy it only to specific sites.
 
-For more information about managing SharePoint Framework solution in the App Catalog visit: [https://docs.microsoft.com/sharepoint/use-app-catalog](https://docs.microsoft.com/sharepoint/use-app-catalog) and [https://docs.microsoft.com/sharepoint/dev/general-development/site-collection-app-catalog](https://docs.microsoft.com/sharepoint/dev/general-development/site-collection-app-catalog).
+For more information about managing SharePoint Framework solution in the App Catalog visit: [/sharepoint/use-app-catalog](/sharepoint/use-app-catalog) and [/sharepoint/dev/general-development/site-collection-app-catalog](/sharepoint/dev/general-development/site-collection-app-catalog).
```
Comment (RedZephyr13) [inline docs/spfx/publish-to-marketplace-after-publishing.md:19]: Let's improve this link.  How about this instead:
`[Manage apps using the Apps site](/sharepoint/use-app-catalog) and [Use the site collection app catalog](/sharepoint/dev/general-development/site-collection-app-catalog).`

I basically took the title of the article to make it linkable text.

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[32666df](https://github.com/dereklh7/sp-dev-docs/commits/32666dfdc2917f327cb2eb8b305be1278556b967)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/spfx/publish-to-marketplace-after-publishing.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/publish-to-marketplace-after-publishing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/publish-to-marketplace-after-publishing?branch=pr-en-us-8172) | [Details](#user-content-3595a6840e3d6c4f051122128d68d2b1-e2c64634)
[docs/spfx/use-aadhttpclient.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-aadhttpclient.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/use-aadhttpclient?branch=pr-en-us-8172) | [Details](#user-content-38dcf3b52fdcdc8ec41d348a773a9911-e2c64634)
[docs/spfx/use-theme-colors-in-your-customizations.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-theme-colors-in-your-customizations.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/spfx/use-theme-colors-in-your-customizations?branch=pr-en-us-8172) | [Details](#user-content-d673714d2bccd161432f37861698a99c-e2c64634)

<a id="3595a6840e3d6c4f051122128d68d2b1-e2c64634"></a>
### [docs/spfx/publish-to-marketplace-after-publishing.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/publish-to-marketplace-after-publishing.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="38dcf3b52fdcdc8ec41d348a773a9911-e2c64634"></a>
### [docs/spfx/use-aadhttpclient.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-aadhttpclient.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="d673714d2bccd161432f37861698a99c-e2c64634"></a>
### [docs/spfx/use-theme-colors-in-your-customizations.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/spfx/use-theme-colors-in-your-customizations.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206141448156025-8172/BuildReport?accessString=bb8a0db575e0f294900249238e785678979f61f4d2e15d6e7a7b9f9da33015e1).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by dereklh7** [comment]
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**